CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unversioned
 
+- Major: Added ability to toggle visibility of Channel Tabs - with keyboard shortcut (default: Ctrl+U). (#2600)
```

Comment:
Update changelog to clarify new UX functionality

```suggestion
- Major: Added ability to toggle visibility of Channel Tabs - This can be done by right-clicking the tab area or pressing the keyboard shortcut (default: Ctrl+U). (#2600)
```